packages/modelviewer.dev/data/docs.json

```diff
@@ -867,8 +867,8 @@
     "Attributes": [
       {
         "name": "variant-name",
-        "htmlName": "variantName",
-        "description": "Selects a model variant by name.",
+        "htmlName": "variant-name",
```

Comment:
Let's stick with our conventions here. Also, might be good to add that the default is `null` and options include variant name string or `null`, in the `default` json.

Reply:
Is the "htmlName" supposed to be visible somewhere in on the documentation page? I don't see it.

Reply:
Done.